crates/shared/src/macros.rs

```diff
@@ -40,6 +40,13 @@ macro_rules! dummy_contract {
     };
 }
 
+#[macro_export]
```

Comment:
I'm not able to tell where this is explicitly used...

Reply:
Its used in `current_block.rs` (a bit discretely - the `eth_call` that we do there needs the bytecode of the `FetchBlock.sol` contract).